interpreter/cling/tools/packaging/cpt.py

```diff
@@ -257,15 +259,20 @@ def download_llvm_binary():
                 llvm_config_path = llvm_config_path[:-1]
             llvm_flags = "-DLLVM_BINARY_DIR={0} -DLLVM_CONFIG={1} -DLLVM_LIBRARY_DIR={2} -DLLVM_MAIN_INCLUDE_DIR={3} -DLLVM_TABLEGEN_EXE={4} \
                           -DLLVM_TOOLS_BINARY_DIR={5} -DLLVM_TOOL_CLING_BUILD=ON".format(llvm_dir, llvm_config_path,
-                          os.path.join(llvm_dir, 'lib'), os.path.join(llvm_dir, 'include'), os.path.join(llvm_dir, 'bin', 'llvm-tblgen'),
-                          os.path.join(llvm_dir, 'bin'))
+                           os.path.join(llvm_dir, 'lib'), os.path.join(llvm_dir, 'include'), os.path.join(llvm_dir, 'bin', 'llvm-tblgen'),
+                           os.path.join(llvm_dir, 'bin'))
+            if args['verbose']:
+                llvm_flags +=  "-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON"
     else:
         raise Exception("Building clang using LLVM binary not possible. Please invoke cpt without --with-binary-llvm and --with-llvm-tar flags")
     if tar_required:
+
         llvm_flags = "-DLLVM_BINARY_DIR={0} -DLLVM_CONFIG={1} -DLLVM_LIBRARY_DIR={2} -DLLVM_MAIN_INCLUDE_DIR={3} -DLLVM_TABLEGEN_EXE={4} \
-                      -DLLVM_TOOLS_BINARY_DIR={5} -DLLVM_TOOL_CLING_BUILD=ON".format(srcdir, os.path.join(srcdir, 'bin', 'llvm-config'),
-                       os.path.join(srcdir, 'lib'), os.path.join(srcdir, 'include'), os.path.join(srcdir, 'bin', 'llvm-tblgen'),
-                       os.path.join(srcdir, 'bin'))
+                          -DLLVM_TOOLS_BINARY_DIR={5} -DLLVM_TOOL_CLING_BUILD=ON".format(llvm_dir, llvm_config_path,
+                           os.path.join(llvm_dir, 'lib'), os.path.join(llvm_dir, 'include'), os.path.join(llvm_dir, 'bin', 'llvm-tblgen'),
+                           os.path.join(llvm_dir, 'bin'))
+        if args['verbose']:
+            llvm_flags += "-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON"
```

Comment:
Likewise.

This code snippet seems to be repeating 3 times. Would it make sense to move it into a utility function?

Reply:
My bad, just added new helper function for setting the llvm_flags

Reply:
@vgvassilev